conda-env/dev.yml

```diff
@@ -0,0 +1,34 @@
+# Conda development environment for testing local source code changes to the `esgfpub` and `warehouse` modules before merging them to production (`master` branch).
```

Comment:
Ok, so this is to support the warehouse _module_, not the warehouse _system_ (the latter includes extraction, validations, postprocess and publication taken together.)

Reply:
To clarify, the intention of this env is to test source code changes to `esgfpub` and/or `warehouse` before they are merged into production (`master` branch).

It supports the same warehouse operations as `prod.yml` (`extract`, `validate`, `postprocess`). I need to update my diagram to reflect this information.